snapshot/src/test/java/io/camunda/zeebe/snapshots/ReceivedSnapshotTest.java

```diff
@@ -191,20 +173,17 @@ public void shouldPersistEvenIfSameChunkIsConsumedMultipleTimes() {
   }
 
   @Test
-  public void shouldReturnAlreadyExistingSnapshotOnPersist() {
+  public void shouldThrowWhenAlreadyExistingSnapshotWasReceivedAgain() {
```

Comment:
ℹ️ Here I was not sure how to handle this case to be honest. I'm open for alternatives.